frontends/election-manager/src/components/election_manager.tsx

```diff
@@ -40,6 +40,7 @@ import { SettingsScreen } from '../screens/settings_screen';
 import { LogsScreen } from '../screens/logs_screen';
 import { ReportsScreen } from '../screens/reports_screen';
 import { smartcardTypeRegex } from '../config/types';
+import { SmartcardModal } from './smartcard_modal';
```

Comment:
You should review this commit using the [`?w=1`](https://github.com/votingworks/vxsuite/pull/2125/commits/116f4a43f54fec3bc7838e54ad454f24247105d0?w=1) param!